pkg/auth/auth.go

```diff
@@ -176,7 +176,7 @@ func (c *ClientConfig) WithEmbeddedToken() (*ClientConfig, error) {
 
 	log.Info("Generating token")
 
-	gen, err := token.NewGenerator()
+	gen, err := token.NewGenerator(true)
```

Comment:
@undeadops Here needs 2 parameters .
```suggestion
      gen, err := token.NewGenerator(true, false)
```